src/test/testAddresses.ts

```diff
@@ -21,7 +21,7 @@ export const generateDsnpUserId = (): HexString => {
  */
 export const getPrefabWalletAddress = (index: number): HexString => {
   const regex = /0/gi;
-  const address = "0x" + "CODE0000".replace(regex, index.toString());
+  const address = "0x" + "CADE0000".replace(regex, index.toString());
```

Comment:
why did these change?

Reply:
They fail validation. O is not a valid hexadecimal character.

Reply:
Ah. I thought it was a zero. #fonts